app/views/docs/permissions.phtml

```diff
@@ -44,60 +44,62 @@
     </thead>
     <tbody>
     <tr>
-        <td>
-            role:all
-            <p class="margin-top-small text-fade">version >= 0.12</p>
-        </td>
+        <td>Role.any()</td>
         <td>Wildcard permission. Gives anyone read or write access.</td>
     </tr>
     <tr>
-        <td>user:[USER_ID]</td>
+        <td>Role.guests()</td>
+        <td>Access to any guest user with an anonymous session. Logged in users don't have access to this role.</td>
```

Comment:
Not sure if true. Anonymous session means he is a logged-in user. Right?

Guest is for when you don't have session.

Reply:
English-specific comment, shouldn't it be `Gives access to any ...`? If yes, apply to all descriptions of permissions.

Reply:
Yep, grant access is better here.